### PR TITLE
Atuin stats with day, month, week and year filter

### PR DESCRIPTION
**Pull request by bahdotsh**
Filter `atuin stats` with day, week, month and year filer by using `atuin stats today`, `atuin stats week`, `atuin stats month`, `atuin stats year`.

### CODE REVIEW

File: src/command/client/stats.rs [hunk 1]
```diff
@@ -82,6 +82,22 @@ impl Cmd {
...
+        } else if words.trim() == "month"  {
+            let end = parse_date_string(&Local::now().date().format("%Y-%m-%d").to_string(), Local::now(), settings.dialect.into())?;
+            let start = end - Duration::days(31);
+            db.range(start.into(), end.into()).await?
+        } else if words.trim() == "week" {
+            let end = parse_date_string(&Local::now().date().format("%Y-%m-%d").to_string(), Local::now(), settings.dialect.into())?;
+            let start = end - Duration::days(7);
+            db.range(start.into(), end.into()).await?
+        } else if words.trim() == "year" {
+            let start = parse_date_string(Local::now().year().to_string().as_str(), Local::now(), settings.dialect.into())?;
```
Comment (ellie) [inline src/command/client/stats.rs:98]: same as above, to avoid the parsing!

**Review by ellie — CHANGES_REQUESTED** [review]
Thanks for the work! Just a few pointers and I think we're then good to go 😊

**Review by ellie — APPROVED** [review]
Thank you!

**Comment by ellie** [comment]
just a `cargo fmt` and we're good to go

**Comment by bahdotsh** [comment]
Apologies for the back and forths for a simple PR!

**Review by ellie — APPROVED** [review]

**Pull request merged**